app/src/responsive.sass

```diff
@@ -1 +1,6 @@
-$breakpoint-tablet: 500px
+//phone-size:  320px and 480px
+$breakpoint-phone: 320px
```

Comment:
Seems really small to me, and [others](https://ux.stackexchange.com/questions/53911/what-is-the-smallest-screen-size-you-design-for).